modules/server-endpoints/language-server/src/main/java/org/wso2/carbon/identity/developer/lsp/debug/runtime/DebugSessionManagerImpl.java

```diff
@@ -256,4 +269,14 @@ private void initializeListeners() {
         DebugListenerConfigurator configurator = new DebugListenerConfigurator(this);
         configurator.configure(interceptionEngine);
     }
+
+    private String getTenantDomain(Session session) {
+
+        List<String> tenantList = session.getRequestParameterMap().get(DAPConstants.JSON_KEY_FOR_TENANT_DOMAIN);
+        if (tenantList != null && !tenantList.isEmpty()) {
+            return tenantList.get(0);
+        }
+        return null;
+    }
+
```

Comment:
I think we can remove the blank line here too.

Reply:
Fixed with https://github.com/wso2-extensions/identity-tools-debugger/pull/17